help_to_heat/frontdoor/routing/backwards_routing.py

```diff
@@ -0,0 +1,30 @@
+from help_to_heat.frontdoor.consts import (
+    country_page,
+    govuk_start_page,
+    unknown_page,
+)
+from help_to_heat.frontdoor.routing.forwards_routing import get_next_page
+
+start_page = country_page
+max_route_length = 100
```

Comment:
This is presumably arbitrary to catch loops? Can we mark it as such? How much higher is this than the actual realistic maximum?

Reply:
sure, at a quick count the realistic maximum is around 30